IPython/nbconvert/filters/highlight.py

```diff
@@ -113,7 +113,7 @@ def _pygments_highlight(source, output_formatter, language='ipython', metadata=N
 
     # If the cell uses a magic extension language,
     # use the magic language instead.
-    if language == 'ipython' \
+    if language in ('ipython', 'ipython3') \
```

Comment:
Looking at the code a few lines below, I think this should accept 'ipython2' as well.